youtube_dl/extractor/nest.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class NestIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?video.nest\.com/clip/(?P<id>)(.mp4)?'
+    _TEST = {
+        'url': 'https://video.nest.com/clip/73ddb6bd57c4485597a76e154a4429ea.mp4',
+        'md5': '7ab4eb6d4c2480be1740cc014a76ee96',
+        'info_dict': {
+            'id': '73ddb6bd57c4485597a76e154a4429ea',
+            'ext': 'mp4',
+            'title': "\"\"",
+            'description': '#caughtonNestCam',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        video_id = self._search_regex(
+            r'https:\/\/video.nest.com\/clip\/(.+?)(\.|")', webpage, 'video_id', fatal=False)
```

Comment:
No need to escape `/` here (in a JS /regexp/, yes), but do escape `.`, and don't overwrite video_id:
```suggestion
        video_id = self._search_regex(
            r'https://video\.nest\.com/clip/(.+?)(?:\.|")', webpage, 'video_id', fatal=False) or video_id
```
Actually, is this ever different from the value extracted from the page URL? With the correct `_VALID_URL`, you should have a good value for it. If you do need to do this search, use the `_VALID_URL` again:
```suggestion
        video_id = self._search_regex(
            self._VALID_URL, webpage, 'video_id', group='id', fatal=False) or video_id
```
Or just
```suggestion
```